WordPress/WordPressUITests/Tests/EditorTests.swift

```diff
@@ -12,26 +12,52 @@ class EditorTests: XCTestCase {
         app.launchArguments = ["NoAnimations"]
         app.activate()
 
+        // Media permissions alert handler
+        systemAlertHandler(alertTitle: "“WordPress” Would Like to Access Your Photos", alertButton: "OK")
+
         editorScreen = LoginFlow
-            .login(email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
-            .tabBar
+            .loginIfNeeded(email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
             .gotoEditorScreen()
     }
 
     override func tearDown() {
         if editorScreen.isLoaded() {
```

Comment:
The tests were consistently crashing here for me. It seems like `tearDown` was sometimes being called at the wrong time? This might be related to moving on from a failing test to the next one.

Checking `editorScreen != nil` before `isLoaded` resolved the issue. YMMV?

Reply:
Huh, I didn't get any crashes there but if it happened with a failing test I probably just didn't get the same failure. I'll work on a fix for that.

Reply:
Yep, I reproduced the crash when the test failed on an unsuccessful login, and checking `editorScreen != nil` first resolved it for me.

This led me to a couple other possible issues with the `tearDown` (depending on where tests fail). I'll work on a flow to exit the editor from any editor screen, to make that `tearDown` more resilient.